src/main/steps/c100-rebuild/safety-concerns/abduction/passport-amount/content.ts

```diff
@@ -0,0 +1,141 @@
+import { YesOrNo } from '../../../../../app/case/definition';
+import { TranslationFn } from '../../../../../app/controller/GetController';
+import { FormContent } from '../../../../../app/form/Form';
+import { atLeastOneFieldIsChecked, isFieldFilledIn, isTextAreaValid } from '../../../../../app/form/validation';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const en = () => ({
+  caption: 'Safety concerns',
+  title: 'Provide details of the children’s passports',
+  childrenMoreThanOnePassport: 'Do the children have more than one passport?',
+  possessionChildrenPassport: 'Who is in possession of the children’s passports?',
+  select_all_relevant: 'Select all that apply',
+  one: 'Yes',
+  two: 'No',
+  option1: 'Mother',
+  option2: 'Father',
+  option3: 'Other',
+  otherDetails: 'Provide more details',
+  guidanceParentalChildHyperlink: 'https://www.gov.uk/government/collections/child-abduction',
```

Comment:
@kumarmadurai We have created a common content.ts & template.njk files for these links.
Please remove these links.

Reply:
fixed